### PR TITLE
Add button to edit data for logged in users

### PR DESCRIPTION
**Pull request by aabdaab**
This PR adds buttons redirecting to corresponding admin panel sections for logged in user.
Check [PT](https://www.pivotaltracker.com/story/show/170851810) to see in which places the button should appear.

manageDataBTN

### CODE REVIEW

**Comment by tsubik** [comment]
Could be useful for TPI side too, pages, news, companies. I'd suggest to do that similarly as in Wordpress. Having fixed top bar and the rest of page should have margin-top. What do you think?

**Comment by tsubik** [comment]
Would be easier as we will only update only one rails template to have this bar. And the link could be provided using variable from controller.

**Comment by tsubik** [comment]
And the user will always know where to expect the link.

**Comment by aabdaab** [comment]
coming back to this :)

**Comment by aabdaab** [comment]
CCLOW fixed navbar:
image

TPI fixed navbar:
image

Not sure about the styles applied so it'd be good to get your opinion about it ☝️ 
The fixed navbar is always visible for logged in users. The title (`Dashboard` on the example) changes based on what page we are in.

File: app/assets/stylesheets/cclow/layout.scss [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 $search-field-height: 3.75rem;
 
+.header__margin {
+  margin-top: var(--fixed-navbar-height);
```
Comment (tsubik) [inline app/assets/stylesheets/cclow/layout.scss]: Nice! I have to use those more often

File: app/controllers/cclow/climate_targets_controller.rb [hunk 2]
```diff
@@ -10,6 +10,8 @@ def index
       add_breadcrumb('Search results', request.path) if params[:q].present? || params[:recent].present?
 
       @climate_targets = Queries::CCLOW::TargetQuery.new(filter_params).call
+      @admin_panel_section_title = 'Climate targets'
```
Comment (tsubik) [inline app/controllers/cclow/climate_targets_controller.rb]: I would probably encapsulate this in some common method, similar way as `add_breadcrumb` is done.

File: app/assets/stylesheets/cclow/layout.scss (hunk 1 above)
Comment (aabdaab) [reply app/assets/stylesheets/cclow/layout.scss]: yeah! It's really powerful

File: app/controllers/cclow/climate_targets_controller.rb (hunk 2 above)
Comment (aabdaab) [reply app/controllers/cclow/climate_targets_controller.rb]: good idea

File: app/assets/stylesheets/cclow/layout.scss (hunk 1 above)
Comment (aabdaab) [reply app/assets/stylesheets/cclow/layout.scss]: very straightforward article about css variables and how to take advantage of them: https://dev.to/nirazanbasnet/understanding-css-variables-4cj0

Comment (tsubik) [reply app/assets/stylesheets/cclow/layout.scss]: Thanks!

File: app/admin/override.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ActiveAdmin::Devise::SessionsController.class_eval do
```
Comment (tsubik) [inline app/admin/override.rb]: could be better to add this to active_admin config file in `initializers`

**Comment by aabdaab** [comment]
done ☝️

**Pull request merged**